src/decoratedtextinput/styles/decoratedtextinput.m.css

```diff
@@ -0,0 +1,18 @@
+@import '../../common/styles/variables.css';
+
+.addon {
```

Comment:
Should we be making any classes used in the base styles `fixed` classes? At least ones that the user wouldn't want to override. Maybe none of these are applicable.

Reply:
I thought about it, since the flexbox styles could be seen as integral to the display. But then I thought another possibility might be someone who wants the addon text to display within the text input box with absolute positioning and padding.

Maybe that's not enough of a common use case to cater to it (or any other weird style variation). I'm not sure, and I'm open to other opinions.